sdk/storage/Azure.Storage.Files.DataLake/src/Models/ChangeAccessControlListPartialResult.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System.Collections.Generic;
+
+namespace Azure.Storage.Files.DataLake.Models
+{
+    /// <summary>
+    /// ChangeAccessControlListPartialResult contains partial counts of operations that change Access Control Lists recursively.
+    /// Additionally it exposes path entries that failed to update while these operations progress.
+    /// See <see cref="DataLakePathClient.SetAccessControlListRecursive(System.Collections.Generic.IList{PathAccessControlItem}, int?, System.IProgress{ChangeAccessControlListPartialResult}, System.Threading.CancellationToken)"/>
+    /// <see cref="DataLakePathClient.SetAccessControlListRecursiveAsync(System.Collections.Generic.IList{PathAccessControlItem}, int?, System.IProgress{ChangeAccessControlListPartialResult}, System.Threading.CancellationToken)"/>
+    /// <see cref="DataLakePathClient.ModifyAccessControlListRecursive(System.Collections.Generic.IList{PathAccessControlItem}, int?, System.IProgress{ChangeAccessControlListPartialResult}, System.Threading.CancellationToken)"/>
+    /// <see cref="DataLakePathClient.ModifyAccessControlListRecursiveAsync(System.Collections.Generic.IList{PathAccessControlItem}, int?, System.IProgress{ChangeAccessControlListPartialResult}, System.Threading.CancellationToken)"/>
+    /// </summary>
+    public class ChangeAccessControlListPartialResult : ChangeAccessControlListResult
```

Comment:
One SDK will probably want to rename this.

Reply:
Also, we need to add a static method to create this type in ```models/DataLakeModelFactory.cs``` so users can construct new instances for mocking.

Reply:
I think maybe we should just add FailedEntries to ```ChangeAccessControlListResult.cs```

Reply:
I was thinking about adding FailedEntries to ChangeAccessControlListResult. However the return value from Set/Modify ACL is never going to contain it. This could result in people asking why is it empty, so I think it's better to be strict about that.
@tg-msft could chime in on naming?